test/bst.t

```diff
@@ -248,52 +248,42 @@ Testing Environment
 
 Verifying cgroup system config
 
-	$ if touch /sys/fs/cgroup/cgroup.procs         ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst                  ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/cgroup.procs         ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/cpu.max          ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/cpu.weight       ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/memory.min       ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/memory.low       ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/memory.high      ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/memory.max       ; then exit 0; else exit 80; fi
-	$ if touch /sys/fs/cgroup/bst/memory.swap.high ; then exit 0; else exit 80; fi
-	$ if touch /sys/fs/cgroup/bst/memory.swap.max  ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/io.weight        ; then exit 0; else exit 80; fi
-	$ if touch /sys/fs/cgroup/bst/io.max           ; then exit 0; else exit 80; fi
-	$ if touch /sys/fs/cgroup/bst/io.latency       ; then exit 0; else exit 80; fi
 	$ if touch /sys/fs/cgroup/bst/pids.max         ; then exit 0; else exit 80; fi
 
 Test --no-cgroup-remount disabling mount over
 
 	$ bst --cgroup /sys/fs/cgroup/bst --no-cgroup-remount cat /sys/fs/cgroup/bst/cgroup.procs
-	0
-	1
-	2
 	$ ! bst --cgroup /sys/fs/cgroup/bst test -f /sys/fs/cgroup/bst/cgroup.procs
 
 Test --share cgroup disabling mount over
 
 	$ bst --cgroup /sys/fs/cgroup/bst --share cgroup cat /sys/fs/cgroup/bst/cgroup.procs
-	0
-	1
-	2
 	$ ! bst --cgroup /sys/fs/cgroup/bst test -f /sys/fs/cgroup/bst/cgroup.procs
 
 Test --share mount disabling mount over
 
 	$ bst --cgroup /sys/fs/cgroup/bst --share mount cat /sys/fs/cgroup/bst/cgroup.procs
-	0
-	1
-	2
 	$ ! bst --cgroup /sys/fs/cgroup/bst test -f /sys/fs/cgroup/bst/cgroup.procs
 
 Test trying to apply climits without specifying a cgroup
 
 	$ ! bst --no-cgroup-remount --limit cpu.max=3333 cat /sys/fs/cgroup/bst/cpu.max
 	bst: unable to apply limits without --cgroup specified: Success
 
-Test cpu.max cgroup usage
+Test overspecified --limits
+	$ ! bst --cgroup /sys/fs/cgroup/bst --limit cpu.max=2222 --limit cpu.max=1111
+	bst: double specified --limit argument: Success
```

Comment:
Haha, weird error! This is caused by the error path calling err rather than errx, since  the former adds strerror(errno) to the message.

Reply:
Switched to errx. It appears --rlimit has this issue as well?

Reply:
Likely. Feel free to fix this in a second PR

Reply:
Will do